### PR TITLE
Con 2595

### PR DESCRIPTION
**Pull request by maddiegos**

### CODE REVIEW

File: helm/charts/hpe-csi-driver/templates/hpe-csi-controller.yaml [hunk 1]
```diff
@@ -64,6 +64,7 @@ spec:
           args:
             - "--v=5"
             - "--csi-address=$(ADDRESS)"
+            - "--timeoout=180"
```
Comment (raunakkumar) [inline helm/charts/hpe-csi-driver/templates/hpe-csi-controller.yaml:67]: No, we don't want it to be hardcoded. it needs to be configurable like cspClientTimeout

**Review by raunakkumar — CHANGES_REQUESTED** [review]

File: helm/charts/hpe-csi-driver/templates/hpe-csi-controller.yaml (hunk 1 above)
Comment (maddiegos) [reply helm/charts/hpe-csi-driver/templates/hpe-csi-controller.yaml:67]: ok

Comment (maddiegos) [reply helm/charts/hpe-csi-driver/templates/hpe-csi-controller.yaml]: modified

Comment (pavansshanbhag) [reply helm/charts/hpe-csi-driver/templates/hpe-csi-controller.yaml]: Have we tested this ? the timeout spelling is incorrect

**Pull request closed without merging**